packages/components/src/components/primitives/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { Breadcrumb } from './Breadcrumb';
+
+export default {
+  component: Breadcrumb,
+  title: 'Components/Primitives/Breadcrumb',
+} as Meta;
+
+const Template = (args) => (
+  <Breadcrumb {...args}>
```

Comment:
Can we change the separator to match the design? It should look like a chevron right.

Also, can we add a changeset for this PR?

Reply:
done!. Pau the separator is handled as a prop and use Icon component, I use one of our Icons.